pymilo/transporters/preprocessing_transporter.py

```diff
@@ -96,7 +113,71 @@ def deserialize_pre_module(self, serialized_pre_module):
         data = serialized_pre_module["pymilo-preprocessing-data"]
         associated_type = SKLEARN_PREPROCESSING_TABLE[serialized_pre_module["pymilo-preprocessing-type"]]
         retrieved_pre_module = associated_type()
-        gdst = GeneralDataStructureTransporter()
+        for key, _ in data.items():
```

Comment:
We can also use the `value` in following `if` statement conditions. If you are not using `value`, you can simply iterate over `data`.

Reply:
good point, sure.